### PR TITLE
Update ceoi-10-AHugeTower.mdx because official solution's explanation doesn't go in-depth

### PR DESCRIPTION
**Pull request by Evang144**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: solutions/ceoi-10-AHugeTower.mdx [hunk 1]
```diff
@@ -2,16 +2,27 @@
...
 source: CEOI 2010
 title: A Huge Tower
-author: Óscar Garries
+author: Óscar Garries, Evan Zhao
 ---
 
 [Official Analysis](http://ceoi2010.ics.upjs.sk/pub/documents/ceoi_summary.pdf#page=14)
 
+## Explanation
+
```
Comment (SansPapyrus683) [inline solutions/ceoi-10-AHugeTower.mdx:11]:
```suggestion

First, let's define some variables.
```

File: solutions/ceoi-10-AHugeTower.mdx [hunk 2]
```diff
@@ -2,16 +2,27 @@
...
+Let $a_1, a_2, ..., a_n$ be the block sizes, sorted from least to greatest (in other words, $a_1 \leq a_2 \leq a_3 \leq ... \leq a_n$.).
```
Comment (SansPapyrus683) [inline solutions/ceoi-10-AHugeTower.mdx:12]:
```suggestion
Let $a_1, a_2, ..., a_n$ be the block sizes, sorted from least to greatest.
```

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

**Review by SansPapyrus683 — APPROVED** [review]
yee

**Review by ryanchou-dev — APPROVED** [review]
first time github contributor!!!!!! 🥳

**Review by TruaShamu — APPROVED** [review]
looks good to me :)

**Pull request merged**